DependencyInjection/DoctrineExtension.php

```diff
@@ -643,6 +643,7 @@ protected function loadOrmEntityManager(array $entityManager, ContainerBuilder $
             'setProxyDir' => '%doctrine.orm.proxy_dir%',
             'setProxyNamespace' => '%doctrine.orm.proxy_namespace%',
             'setAutoGenerateProxyClasses' => '%doctrine.orm.auto_generate_proxy_classes%',
+            'setSchemaIgnoreClasses' => '%doctrine.orm.schema_ignore_classes%',
```

Comment:
Should we really have a single list of ignored classes common for all entity managers ? Shouldn't we have separate list per managers instead ? This is not a configuration mapping to a global state

Reply:
I'm not sure it's possible since the `doctrine.orm.configuration` is global and not per EM 🤔

Reply:
really ? This looks weird to me...

Reply:
Is it this? https://github.com/doctrine/DoctrineBundle/blob/4f4387cbbd71a3a5037e44cefa466734499e1e57/DependencyInjection/DoctrineExtension.php#L608